### ISSUE TITLE
Talk : Designing Deep Learning Pipelines For Visual Perception  

### ISSUE THREAD

**Issue by aradhyamathur** [mohitsharma29/ML-Reading-Group#4]
We will look at what occurs under the hood of modern AI based image processing in camera systems and try to gain a deeper insight what sort of mathematical intuition is needed when designing for a particular kind of visual perception system. 
Modern cameras have a lot of new features and filters and thus we will in particular be focusing on the image colourisation and stylization based approaches that have been widely used and adopted. Further we will focus on the mathematical intuition behind different approaches by focusing on different types of loss functions and their impacts and the process in general.

Prerequisites
Basic understanding of deep convolutional neural networks and GANs.
For introduction to CNNs you can refer [here](https://towardsdatascience.com/an-introduction-to-convolutional-neural-networks-eb0b60b58fd7)
For introduction to GANs you can refere [1](https://machinelearningmastery.com/what-are-generative-adversarial-networks-gans/), [2](https://arxiv.org/abs/1701.00160)

**Comment by aradhyamathur** [comment]
Slides can be found [here](https://docs.google.com/presentation/d/1WwVG_0tIi3kOBygoH2FP2IIV1zhoSvwFKopJ0vfRDJY/edit?usp=sharing)
